### ISSUE TITLE
Can't remove rel="noopener noreferrer nofollow" from inline links

### ISSUE THREAD

**Issue by svale** [verbb/vizy#159]
Hi!

We are seeing an issue where inline links in the Vizy text editor are stuck with a `rel="noopener noreferrer nofollow"` attribute. This is reasonable (though perhaps preferable with just `noopener noreferrer`) for links opening in new windows. However it is unfortunate for other (and especially internal) links. 

Internal links could be an entry or plain link format, but  either way they should not have the `rel`-attribute if not target is not `_blank`. They also seem to get stuck with an empty `target`-attribute. 

<img width="704" alt="image" src="https://user-images.githubusercontent.com/5339180/176376351-73288f6f-ebdb-431a-a8c7-ba89408d4068.png">

**Steps to reproduce**
1. Add an inline link in Vizy text editor
3. Check html source

**Additional info**
- Plugin version: v1.0.13
- Craft version: 3.7.39
- Multi-site: No

**Additional context**
Perhaps the issue could be resolved by more actively unsetting the attribute in https://github.com/verbb/vizy/blob/58a30d991a8fde69682382e36ecd958febe15262/src/marks/Link.php#L22
E.g:
```php
       if (isset($this->attrs['target']) && $this->attrs['target'] === '_blank') {
            $this->attrs['rel'] = 'noopener noreferrer nofollow';
        } else {
          unset($this->attrs['rel'])
        }
```

Thank you for providing fantastic plugins!

**Comment by engram-design** [comment]
Not quite sure why I added `nofollow`. But yes, it should only be applying this for links that have a new window option set. Looks like this is a Tiptap inherited thing. But also, to be fair this is only in the editor, not when we output the content of the field via Twig.

Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.15",
  "...": "..."
}
```

Then run `composer update`.
